docker/deployment/docker-compose.yml

```diff
@@ -47,9 +46,9 @@ services:
           devices:
             - capabilities: [gpu]
     command: sleep infinity
-    network_mode: "host"
 
   mosquitto:
     image: eclipse-mosquitto
     command: mosquitto -c /mosquitto-no-auth.conf
-    network_mode: "host"
+    ports:
+      - "18883:1883"
```

Comment:
is the extra `8` intended @leandropineda ?

Reply:
Yes, just to avoid issues with a broker running on the default port.